packages/misc/src/lib/misc.ts

```diff
@@ -666,25 +665,20 @@ export const defaultMintClaimCallback: MintCallback<
   network: string = 'cayenne'
 ): Promise<string> => {
   try {
-    let relayUrl: string = '';
-
-    switch (network) {
-      case LitNetwork.Cayenne:
-        relayUrl = RELAY_URL_CAYENNE + '/auth/claim';
-        break;
-      case LitNetwork.Habanero:
-        relayUrl = RELAY_URL_HABANERO + 'auth/claim';
-        break;
-      case LitNetwork.Manzano:
-        relayUrl = RELAY_URL_MANZANO + 'auth/claim';
-        break;
-      case LitNetwork.DatilDev:
-        relayUrl = RELAY_URL_DATIL_DEV + '/auth/claim';
-        break;
+    const AUTH_CLAIM_PATH = '/auth/claim';
+
+    const relayUrl: string =
+      RELAYER_URL_BY_NETWORK[network as LIT_NETWORK_VALUES];
```

Comment:
1. Do we actually need to accept a string here instead of restricting the function to a specific string union of network keys (maybe `LIT_NETWORK_VALUES`?)?
3. If so, rather than coerce a possibly-invalid value to a type it may not be, then throw an error farther down based on a side-effect of that being invalid, it would be nice to get an explicit error back as a user of this API _and_ get type safety without a hard coercion on the data.  See this pattern here for the type-safe way to handle this using an assertion runtime function that throws a specific error if the data provided can't be coerced to that type.

https://github.com/LIT-Protocol/js-sdk/blob/304dca5a739d38b5a73a3c92715ed4878e6b04c4/packages/wrapped-keys/src/lib/service-client/utils.ts#L42

4. I think this code will no longer use the `params.relayUrl` when it is provided by the caller, because the ternary was removed on L687 (L679 in new code) -- I think for `custom` networks this should still be respected?